lib/network.go

```diff
@@ -4774,6 +4775,161 @@ type TransactionSpendingLimit struct {
 	DAOCoinLimitOrderLimitMap map[DAOCoinLimitOrderLimitKey]uint64
 }
 
+func (tsl *TransactionSpendingLimit) ToMetamaskString(params *DeSoParams) string {
+	var str string
+	var indentationCounter int
+	_indt := _computeIndentation
```

Comment:
can't we just rename the function define below? I don't love the idea of renaming a function inside another

Reply:
yup, makes sense